plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java

```diff
@@ -1079,6 +1089,29 @@ public void run()
             refreshEntityAndAttributes();
         }
     }
+    
+    private class ChangeERDNotationStyleAction extends Action {
+        private final ERDNotationDescriptor notation;
+
+        public ChangeERDNotationStyleAction(ERDNotationDescriptor notation) {
+            super(notation.getName(), AS_CHECK_BOX);
+            this.notation = notation;
+        }
+
+        @Override
+        public boolean isChecked() {
+            if (notation == null || getDiagram().getDiagramNotation() == null) {
+                return false;
+            }
+            return notation.getId().equals(getDiagram().getDiagramNotation().getId());
+        }
+
+        @Override
+        public void run() {
+            getDiagram().setDiagramNotation(notation);
+            refreshDiagram(true, true);
```

Comment:
Can we avoid refreshing the diagram layout, or is it needed? We can update entities and relations separately.

Reply:
@Destrolaric ?